tests/Utils/TestRunner.hs

```diff
@@ -72,19 +72,8 @@ startServer configFile = do
   return (asyncServer, asyncCmd, asyncHist)
 
 stopServer :: (Async (), Async (), Async ()) -> IO ()
-stopServer (asyncServer, asyncCmd, asyncHist) = do
-  uninterruptibleCancel asyncCmd
-  uninterruptibleCancel asyncHist
-  uninterruptibleCancel asyncServer
-  mapM_ checkFinished [asyncServer, asyncCmd, asyncHist]
-  where checkFinished asy = do
-          asyPoll <- poll asy
-          case asyPoll of
-            Nothing -> Exception.evaluate $ error $ ("Thread " ++
-                                                      (show (asyncThreadId asy)) ++
-                                                      " could not be cancelled.")
-            _ -> return ()
-          
+stopServer (asyncServer, _, _) = cancel asyncServer
```

Comment:
If @slpopejoy signs off on this, looks like it's good to go.

Reply:
@LindaOrtega made this suggestion originally. This change prevents a few sporadic connection failures in the tests.